westonganger/active_record_simple_execute
# Refractor CI, Add Rails 7.1 and Ruby 3.2

- Refractor CI Github workflow
- Add Rails 7.1 to test matrix
- Add Ruby 3.2 to test matrix
- Remove some development dependencies in gemspec
- Remove ruby version check from gemspec
- Update gitignore list for sqlite DB files
